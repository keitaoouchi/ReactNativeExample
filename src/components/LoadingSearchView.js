import React from "react";
import {ActivityIndicator, View} from "react-native";
import Color from "../misc/Color";
import Workaround from "../misc/Workaround";

export default LoadingSearchView = () => (
  <View style={ style.root }>
    <ActivityIndicator
      animating={true}
      color="white"
      size="large"
    />
  </View>
);

const style = {
  root: {
    flex: 1,
    flexDirection: 'column',
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: Color.black,
    ...Workaround.root,
  },
};