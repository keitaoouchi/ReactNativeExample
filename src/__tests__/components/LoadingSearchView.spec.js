import "react-native";
import React from "react";
import renderer from "react-test-renderer";
import LoadingSearchView from "../../../src/components/LoadingSearchView";

describe('LoadingSearchView', () => {

  it('renders correctly', () => {
    const tree = renderer.create(
      <LoadingSearchView />,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});