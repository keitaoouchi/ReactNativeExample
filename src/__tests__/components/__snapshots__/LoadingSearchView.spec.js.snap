// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingSearchView renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#181818",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="white"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
